sync.sh

```diff
@@ -64,8 +72,8 @@ function sync_data {
         doc_version=$(echo "${path}" | cut -d'/' -f7 | cut -d'=' -f2)
 
         
-        namespace_dir="${data_path}/${namespace}"
-        filename="${submission_date}.${doc_type}.${doc_version}.batch.json"
+        namespace_dir="${data_path}/${submission_date}/${namespace}"
+        filename="${doc_type}.${doc_version}.batch.json"
```

Comment:
I'm changing the file-structure here, I'll probably tag this as a new release afterwards.